CMakeLists.txt

```diff
@@ -267,10 +261,10 @@ if(AMENT_ENABLE_TESTING)
   ament_lint_auto_find_test_dependencies()
 endif()
 
-  file(
-    COPY "${PROJECT_NAME}"
-    DESTINATION "${CMAKE_CURRENT_BINARY_DIR}"
-  )
+file(
+  COPY "${PROJECT_NAME}"
```

Comment:
What is being installed here? If it is the library the `file(TARGET` signature should be used.